### PR TITLE
EN-1857-read-and-save-metadata-on-images-in-multiple-uploader

### PR DESCRIPTION
**Pull request by alepintus**
Hey @joewhite101 @eugeniosant ,
This commit is for resource metadata task.

It should be merged a the same time with the entando-components and entando-core PR below:

https://github.com/entando/entando-core/pull/568
https://github.com/entando/entando-components/pull/304

Thanks!

### CODE REVIEW

File: entando-sample/src/main/resources/archetype-resources/src/test/config/conf/systemParams.properties [hunk 1]
```diff
@@ -49,8 +49,12 @@ algorithm.securityConfiguration.path=target/test/conf/security.properties
 
 jacms.frontend.userfilter.dateFormat=yyyy-MM-dd
 
+jacms.publicContentSearcher.forceCaseInsensitiveLikeSearch=false
```
Comment (eugeniosant) [inline entando-sample/src/main/resources/archetype-resources/src/test/config/conf/systemParams.properties]: the *forceCaseInsensitiveLikeSearch parameters are no longer used. To remove

File: web-app-bpm/src/main/resources/archetype-resources/src/main/config/systemParams.properties [hunk 2]
```diff
@@ -49,8 +49,12 @@ algorithm.securityConfiguration.path=/WEB-INF/conf/security.properties
 
 jacms.frontend.userfilter.dateFormat=yyyy-MM-dd
 
+jacms.publicContentSearcher.forceCaseInsensitiveLikeSearch=true
+jacms.workContentSearcher.forceCaseInsensitiveLikeSearch=true
```
Comment (eugeniosant) [inline web-app-bpm/src/main/resources/archetype-resources/src/main/config/systemParams.properties]: the *forceCaseInsensitiveLikeSearch parameters are no longer used. To remove

File: web-app-bpm/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/conf/systemParams.properties [hunk 3]
```diff
@@ -37,8 +37,12 @@ page.charset.allowedValues=iso-8859-1,utf-8
 
 jacms.frontend.userfilter.dateFormat=yyyy-MM-dd
 
+jacms.publicContentSearcher.forceCaseInsensitiveLikeSearch=true
```
Comment (eugeniosant) [inline web-app-bpm/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/conf/systemParams.properties]: the *forceCaseInsensitiveLikeSearch parameters are no longer used. To remove

File: web-app-bpm/src/main/resources/archetype-resources/src/test/config/conf/systemParams.properties [hunk 4]
```diff
@@ -49,8 +49,12 @@ algorithm.securityConfiguration.path=target/test/conf/security.properties
 
 jacms.frontend.userfilter.dateFormat=yyyy-MM-dd
 
+jacms.publicContentSearcher.forceCaseInsensitiveLikeSearch=false
```
Comment (eugeniosant) [inline web-app-bpm/src/main/resources/archetype-resources/src/test/config/conf/systemParams.properties]: the *forceCaseInsensitiveLikeSearch parameters are no longer used. To remove

**Comment by alepintus** [comment]
Hey @eugeniosant ,
Thanks for the review! 
forceCaseInsensitiveLikeSearch properties removed as suggested from all systemParams.properties.

**Pull request merged**